README.md

```diff
@@ -23,4 +23,6 @@ This macro DOES NOT support inline HTML. This functionality was removed due to i
 Create a new release based on the correct release branch and correct tag name.
 
 - release/1.6.x will target Confluence < 9. Eg create a release 1.6.30 with tag 1.6.30 on release/1.6.x
-- main will target Confluence >= 9.0, and should be tagged with 2.x.y. Eg create a release 2.0.0 with tag 2.0.0 on main
+- release/2.x will target Confluence 9, and should be tagged with 2.x.y. Eg create a release 2.0.0 with tag 2.0.0 on release/2.x
+- main will target Confluence 10, and should be tagged with 3.x.y. Eg create a release 3.0.0 with tag 3.0.0 on main
```

Comment:
you'll need to update https://github.com/Atlas-Authority/MarkdownMacro/blob/main/.github/workflows/release.yml#L32 to reflect this new workflow